src/SQL_BAIN_OF_MY_EXISTENCE.sql

```diff
@@ -19,7 +19,7 @@ ADD CONSTRAINT PRIMARY KEY(ID);
 DROP TABLE IF EXISTS Hist_Avg;
 CREATE TABLE Hist_Avg
 SELECT batter, SUM(Hit)/NULLIF (SUM(atBat),0) as Batting_Avg
```

Comment:
very wise to make use of NULLIF to avoid divide by 0 error.

Reply:
Why not use CREATE TABLE IF NOT EXISTS, instead of drop table?

Reply:
very good point.